Packs/TIM_Processing/Playbooks/playbook-TIM_-_Indicator_Relationships_Analysis_README.md

```diff
@@ -0,0 +1,47 @@
+This playbook is designed to assist with a security investigation by providing an analysis of indicator relationships. The following information is included:
+- Indicators of compromise (IOCs) related to the investigation.
+- Attack patterns related to the investigation.
+- Campaigns related to the investigation.
+- IOCs associated with the identified campaigns.
+- Reports containing details on the identified campaigns.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+This playbook does not use any integrations.
+
+### Scripts
+* SearchIndicatorRelationships
+* Set
+
+### Commands
+This playbook does not use any commands.
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| Indicator | Input indicator to analyze its relationships. |  | Optional |
+| LimitResults | The number of results to return. If the input is empty, the limit will be 20. | 200 | Optional |
```

Comment:
If the limit is empty, then won't the default be used?  The default is 200, not 20. Or do I misunderstand?